src/strategies/holds-tokens/examples.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "holds-tokens",
+      "params": {
+        "tokenAddresses": [
+          {"address": "0x7Bd29408f11D2bFC23c34f18275bBf23bB716Bc7", "network": "1"},
+          {"address": "0xc34cbca32e355636c7f52dd8beab0af2396ebd79", "network": "137"}
+        ],
+        "decimals": 0,
+        "snapshot": "latest"
```

Comment:
Please change this to a block number from past

Reply:
why should this be a block number? This is not the snapshot that's an input into the strategy, rather indicating whether we always want to pull the latest snapshot or not. 

If you see code below:

```
let blockTag = chainBlocks[network]
if (options.snapshot === 'latest') {
      blockTag = chainProvider.getBlockNumber();
    }
```

Otherwise, we use the blockTag from the snapshot that was passed through the strategy function: